Source/EasyNetQ/Producer/Rpc.cs

```diff
@@ -171,13 +171,15 @@ protected void RegisterErrorHandling<TResponse>(Guid correlationId, Timer timer,
                             autoDelete: true);
 
                 var exchange = DeclareRpcExchange(conventions.RpcResponseExchangeNamingConvention(responseType));
-
-                advancedBus.Bind(exchange, queue, queue.Name);
+                if (exchange != Exchange.GetDefault())
+                {
+                    advancedBus.Bind(exchange, queue, queue.Name);
```

Comment:
We have two binds in RPC: during requesting and responding. But only one of binds guarded against the default exchange and it looks like a magic knowledge.

What do you think about moving `advancedBus.Bind(exchange, queue, queue.Name);` with check of default exchange to `BindRpcExchangeToQueue` and to use them in request's and reponse's binds?